### PR TITLE
Draw shapes (Triangle Fill and Circle FIll)

### PR DESCRIPTION
**Pull request by iamrakesh28**
Triangle fill and circle fill using bfs is implemented.

### CODE REVIEW

**Comment by iamrakesh28** [comment]
@tymefighter  review this.

**Comment by tymefighter** [comment]
I'll try to do by today 👍

**Comment by tymefighter** [comment]
Great, Now I'll look and review it !

**Comment by tymefighter** [comment]
Wait, did you write those files ?

**Comment by tymefighter** [comment]
Visualizer ??

**Review by tymefighter — APPROVED** [review]
Great Algorithms !
It works perfect !

**Comment by tymefighter** [comment]
@sakshi1603 This can be merged !

File: src/processing/shape/LineDrawer.java [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import java.util.ArrayList;
 import processing.utility.*;
+import infrastructure.validation.logger.LoggerFactory;
+import infrastructure.validation.logger.ILogger;
+import infrastructure.validation.logger.LogLevel;
+import infrastructure.validation.logger.ModuleID;
```
Comment (RathoreDevansh08) [inline src/processing/shape/LineDrawer.java:8]: `@reviewer` ??

Comment (iamrakesh28) [reply src/processing/shape/LineDrawer.java:8]: Reviewer should be me. It's Ahmed file, I will tell him to add

**Comment by himanshu2047** [comment]
Exception handling is not here in any of the file

**Pull request closed without merging**